CI/travis.osx.after_success.sh

```diff
@@ -34,19 +34,21 @@ if [[ "${MUDLET_VERSION_BUILD}" == -ptb* ]]; then
   public_test_build="true"
 fi
 
-# we deploy only certain builds
+# deploy is set to "deploy" on all Github actions. Is there a case where we would run this but not deploy?
 if [ "${DEPLOY}" = "deploy" ]; then
 
   # get commit date now before we check out an change into another git repository
   COMMIT_DATE=$(git show -s --pretty="tformat:%cI" | cut -d'T' -f1 | tr -d '-')
   YESTERDAY_DATE=$(date -v-1d '+%F' | tr -d '-')
 
-  git clone https://github.com/Mudlet/installers.git "${BUILD_DIR}/../installers"
+  # CHANGEME - setting this to my repo to test.
+  git clone https://github.com/rparet/installers.git "${BUILD_DIR}/../installers"
 
   cd "${BUILD_DIR}/../installers/osx"
 
   # setup macOS keychain for code signing on development builds only,
   # as CI's don't allow signing on usual PR builds
+  # I don't think the above comment is accurate at the moment.
```

Comment:
Comment for discussion. Certainly the PR builds are code signed. I think this is trying to say "Setup keychain for code signing with our certificate on *release* builds only".

Reply:
I think we'd want to sign all builds possible, but we can't as that would mean exposing the signing certificate & secret to it to third party forks and it could be leaked that way. At least that's my understanding - I didn't find Githubs explanation very clear here.